src/models.rs

```diff
@@ -76,6 +76,24 @@ impl TimeEntry {
     }
 }
 
+impl Default for TimeEntry {
+    fn default() -> Self {
+        let start = Utc::now();
+        Self {
+            id: -1,
```

Comment:
`id` is irrelevant / shouldn't be in payloads for creating new entries, but the hassle of making the `id` field  `Option` or having some kind of `NewTimeEntry` struct didn't seem worth all the hassle it would introduce.

Reply:
Yeah the existing continue function actually passes the original time-entry's id but it's thankfully ignored by the API